src/main/resources/application.properties

```diff
@@ -108,6 +108,8 @@ csrf.disable=true
 sparql.endpoint=http://virtuoso.ohdsi.org:8890/sparql?default-graph-uri=&query=
 
 security.defaultGlobalReadPermissions=${security.defaultGlobalReadPermissions}
+#security.defaultRoles=${security.defaultRoles}
+security.defaultRoles=public,read restricted Atlas Users
```

Comment:
...and then we would also add role 'shared artifacts reader' (role 16) to this list, based on https://github.com/OHDSI/Atlas/pull/2929